test/integration/members.test.js

```diff
@@ -89,4 +103,38 @@ describe('Members', function () {
         })
     })
   })
+
+  describe('PATCH /members/moveToMembers', function () {
+    before(async function () {
+      await cleanDb()
+      const userId = await addUser(superUser)
+      jwt = authService.generateAuthToken({ userId })
+    })
+    it('Should make the user a member', function (done) {
+      chai
+        .request(app)
+        .patch(`/members/moveToMembers/${superUser.username}`)
```

Comment:
Here we are generating cookie for ankush which makes sense
However we are also updating him only, can we try updating someone else?

Currently you can use any user, no user in the fixtures is having the role `member`